cmd/rook/mon.go

```diff
@@ -71,6 +73,11 @@ func startMon(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("missing mon port")
 	}
 
+	if err := compareMonSecret(clusterInfo.MonitorSecret, path.Join(cfg.dataDir, monName)); err != nil {
+		fmt.Fprintln(os.Stderr, err)
```

Comment:
should we be writing this to the termination log also? https://kubernetes.io/docs/tasks/debug-application-cluster/determine-reason-pod-failure/#writing-and-reading-a-termination-message.  that way the termination message would make it to the pod status visible with `kubectl`.

Also consider adding a new section to the common problems page and providing a link in this error message to that page.

Reply:
sweet, good idea